src/scripts/modal.js

```diff
@@ -0,0 +1,29 @@
+// Get the modal
+var modal =
```

Comment:
Suggestion :- In react we shouldn't be using browser api like getElementByID , since react is declarative , with these code it becomes imperative , because you are directly changing the DOM